.rubocop.yml

```diff
@@ -3,6 +3,10 @@ inherit_from:
 
 AllCops:
   TargetRubyVersion: 2.5
+  SuggestExtensions: false
+  Exclude:
+    - "bin/**/*"
+    - "dennis-client.gemspec"
```

Comment:
I bumped rubocop, otherwise we got an error saying it didn't recognize some cops from the inherited file at https://dev.k.io/rubocop/rubocop.yml

I didn't want to fix those generated files, so just set to exclude them.